zendesk/user.go

```diff
@@ -116,6 +116,7 @@ type UserAPI interface {
 	SearchUsers(ctx context.Context, opts *SearchUsersOptions) ([]User, Page, error)
 	GetManyUsers(ctx context.Context, opts *GetManyUsersOptions) ([]User, Page, error)
 	GetUsers(ctx context.Context, opts *UserListOptions) ([]User, Page, error)
+	GetUsersByGroupID(ctx context.Context, groupID int64, opts *UserListOptions) ([]User, Page, error)
```

Comment:
This is still using the old OBP `UserListOptions` and this function shouldn't be exist, it should be replaced by the newly generated `GetGroupUsersIterator` function.